Add a libtrixi cmake module (#132)
trixi-framework/libtrixi: README.md

```
to build `simple_trixi_controller_f`.

#### CMake

A CMake module for the discovery of an installed libtrixi library is provided with

> [`cmake/FindLibTrixi.cmake`](cmake/FindLibTrixi.cmake). Upon invocation of
> `find_package(LibTrixi)`, the CMake variable `LIBTRIXI_PREFIX` must be set to
> `<install_directory>`. An example `CMakeLists.txt` can be found in

Reviewer: ```suggestion
[`cmake/FindLibTrixi.cmake`](cmake/FindLibTrixi.cmake). Before calling
`find_package(LibTrixi)`, the CMake variable `LIBTRIXI_PREFIX` must be set to
`<install_directory>`. An example `CMakeLists.txt` can be found in
```
Maybe this makes it clearer? To me, "upon invocation ..." sounds a little like "when it is invoked, it will do this and that", whereas you want to express that this needs to be defined beforehand (IIRC).

> [`examples/CMakeListsExternal.txt`](examples/CMakeListsExternal.txt).

Reviewer: ```suggestion
[`examples/external/CMakeLists.txt`](examples/external/CMakeLists.txt).
To see the commands required to build an example program with this CMake project,
please refer to [`examples/external/build.sh`](examples/external/build.sh).
```
